Leanplum/Utils/LPErrorHelper.h

```diff
@@ -0,0 +1,20 @@
+//
+//  LPErrorHelper.h
+//  Leanplum
+//
+//  Created by Grace on 5/7/19.
+//  Copyright © 2019 Leanplum. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface LPErrorHelper : NSObject
+
++ (NSDictionary *)makeUserInfoDict:(NSDictionary *)responseDict;
```

Comment:
If this method is not used and used only internally by LPErrorHelper, lets not expose it.  Please remove it from the header file.